Community/iot_device_registration/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
```

Comment:
Please change all BlueCat copyright dates to 2019. Also in iotdeviceregistration_page.css, iotdeviceregistration_page.js, and iotdeviceregistration_page.html.

Reply:
Copyright dates changed